locales/templates/app.pot

```diff
@@ -1123,3 +1167,44 @@ msgid ""
 "screenshots’ categories."
 msgstr ""
 
+#. Message shown in Home Screen with an icon. Users can click on the icon and
+#. click either they want this feature or not. We'll evaluate if we want to
+#. implement this feature base on the users' feedback.
+msgctxt "exp_survey_wifi"
+msgid ""
+"We plan to integrate “Free Wifi-Finder” to help you find quality free "
+"internet connection nearby.\n"
+"\n"
+" Interested?"
+msgstr ""
+
+#. Message shown in Home Screen with an icon. Users can click on the icon and
+#. click either they want this feature or not. We'll evaluate if we want to
+#. implement this feature base on the users' feedback.
+msgctxt "exp_survey_vpn"
+msgid ""
+"We plan to integrate “VPN service” to get you more freedom and more "
+"privacy when browsing.\n"
+"\n"
+" Interested?"
+msgstr ""
+
+#. Message shown after the user clicks "YES, PLEASE" or "NOPE" on the survey.
+#. %1$s will be replaced with an smiling face with smiling eyes Emoji (U+1F60A)
+#, c-format
+msgctxt "exp_survey_thanks"
+msgid "Thank you for giving us feedback %1$s"
+msgstr ""
+
+#. Text on survey feedback button. This means the user want to have this
+#. feature.
+msgctxt "exp_survey_yes"
+msgid "YES, PLEASE"
+msgstr ""
+
+#. Text on survey feedback button. This means the user don't want this
+#. feature.
```

Comment:
Add a comment explaining that "nope" is just an informal way to say "no". That's not really obvious for someone who doesn't speak English frequently.